src/topology/instances/real.lean

```diff
@@ -2,28 +2,13 @@
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro
-
-The real numbers ℝ.
-
-They are constructed as the topological completion of ℚ. With the following steps:
-(1) prove that ℚ forms a uniform space.
-(2) subtraction and addition are uniform continuous functions in this space
-(3) for multiplication and inverse this only holds on bounded subsets
-(4) ℝ is defined as separated Cauchy filters over ℚ (the separation requires a quotient construction)
-(5) extend the uniform continuous functions along the completion
-(6) proof field properties using the principle of extension of identities
-
-TODO
-
-generalizations:
-* topological groups & rings
-* order topologies
-* Archimedean fields
-
 -/
 import topology.metric_space.basic
 import topology.algebra.uniform_group
 import topology.algebra.ring
+/-!
+# Topological properties of ℝ
```

Comment:
In general, we use `###` for headers inside files.

Reply:
This is a module docstring, so we use `#`.